src/ucp/core/ucp_context.c

```diff
@@ -128,7 +128,8 @@ static ucs_config_field_t ucp_config_table[] = {
   {"MEMTYPE_REG_WHOLE_ALLOC_TYPES", "",
    "Memory types which have whole allocations registered.\n"
    "Allowed memory types: cuda, rocm, rocm-managed",
-   ucs_offsetof(ucp_config_t, ctx.reg_whole_alloc), UCS_CONFIG_TYPE_STRING_ARRAY},
+   ucs_offsetof(ucp_config_t, ctx.reg_whole_alloc_bitmap),
+   UCS_CONFIG_TYPE_BITMAP(ucs_memory_type_names)},
```

Comment:
need to add entry to `ucs_memory_type_names` to solve test issue:
`[UCS_MEMORY_TYPE_LAST + 1] = NULL`